docs/known_issues.rst

```diff
@@ -146,11 +146,6 @@ Workarounds include moving the units outside of the call to
     >>> np.arange(0, 10, 1) * u.m
     <Quantity [0., 1., 2., 3., 4., 5., 6., 7., 8., 9.] m>
 
-Also, `~numpy.linspace` does work:
```

Comment:
I'm confused - why do we want to remove this? It is to give an alternative to using `np.arange`, which does not work with units.

Reply:
Sounds good! Might as well fix that since it clearly confused you, so will likely confuse others...